packages/@azure/arm-batch/tsconfig.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
Why those files show as removed? For example batchai shows as moved instead of removed.

Reply:
in git there isn't really a such thing as a move everything is a remove and add but some git/github UI's try to make it look like a move when there they can find the exact file that was moved. I suspect with the number of tsconfig.json files in this PR git got confused and so it just showed as an add/remove. If you view just this one commit it shows up correctly https://github.com/Azure/azure-sdk-for-js/pull/2194/commits/fb85473a5fd212f36cfc1af681521aaea96f7c1a#diff-8532013f63461117fcc19e468356714c.